README.md

```diff
@@ -83,6 +84,42 @@ GLHF! <3
 
 GLHF! <3
 
+## Tantangan
+
+Anda mempunyai satu bulan penuh pada bulan November untuk membuat sebuah permainan *bebas* dengan bertemakan **peretasan, modifikasi dan/atau penggandaan**.
+
+Apa yang kita maksud dengan **bebas** berdasarkan pada peretasan, modifikasi dan/atau penggandaan? Kami sebenarnya bermaksud, *bebas* yang disarankan. Beberapa contoh mungkin seperti ini:
+
+* seorang pelari yang tak berujung dimana anda *retas* pohon biner menurun di jalur anda dengan kapak yang berpiksel,
```

Comment:
- I think it's more accurate `Game berlari tanpa ujung...`
- Also for more clarity, we should say original term, `di mana anda menebang (*hack*)`
- Word `menurun` maybe could be changed to `pohon biner di sepanjang jalur anda`
- `dengan kapak piksel`